src/util/threadMessage.js

```diff
@@ -229,19 +242,26 @@ async function multipleThreadResults(threadID, message, threads, createMode) {
 /**
  * Sends a message displaying an array of threads with pages.
  *
- * @param {Discord.RichEmbed} templateEmbed Template containing Title,
+ * @param {Discord.RichEmbed} args.templateEmbed Template containing Title,
  * Description, and color that the pages will follow
- * @param {Discord.Message} message The invoking message.
- * @param {Thread} threads An array containing all the threads to display.
- * @param {Number?} itemsPerPage The number of items to display on each page.
+ * @param {Discord.Message} args.message The invoking message.
+ * @param {Thread} args.threads An array containing all the threads to display.
+ * @param {Number?} args.itemsPerPage The number of items to display on each
+ * page. Defaults to 5.
+ * @param {Number?} args.keepalive Milliseconds to keep the message open.
+ * Defaults to 60000.
  *
  * @return {Promise<{
  *  type: 'message' | 'collector',
  *  message: Discord.Message?,
  *  collector: Discord.ReactionCollector?,
  * }>}
  */
-async function pagination(templateEmbed, message, threads, itemsPerPage = 5) {
+async function pagination(args) {
+  const { templateEmbed, message, threads } = args;
+  const itemsPerPage = args.itemsPerPage || 5;
+  const keepalive = args.keepalive || 60000;
```

Comment:
if you wanted to keep everything on the same line you can do
```suggestion
  const { templateEmbed, message, threads, itemsPerPage = 5, keepalive = 60000 } = args;
```

from [Mozilla docs on destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Default_values_2)